pep-0411.txt

```diff
@@ -4,13 +4,21 @@ Version: $Revision$
 Last-Modified: $Date$
 Author: Nick Coghlan <ncoghlan@gmail.com>,
         Eli Bendersky <eliben@gmail.com>
-Status: Active
+Status: Obsolete
```

Comment:
We don't have an Obsolete header. You could perhaps also add `Superseded-By: 2`.
```suggestion
Status: Superseded
```